### PR TITLE
fix:

### PR DESCRIPTION
**Pull request by RomanVolosiuk**
TokenController return
CredentialsDTO class
CredentialsUtils method

### CODE REVIEW

File: src/main/java/ua/volosiuk/mytokenservice/entity/User.java [hunk 1]
```diff
@@ -10,6 +10,6 @@ public class User {
     private Long id;
     private String username;
     private  String password;
-    private boolean enabled;
+    private boolean isEnabled;
```
Comment (mukonin) [inline src/main/java/ua/volosiuk/mytokenservice/entity/User.java:13]: pls revert to "enabled"

File: src/main/java/ua/volosiuk/mytokenservice/controller/TokenController.java [hunk 2]
```diff
@@ -4,18 +4,18 @@
...
 
-    @PostMapping("/token")
+    @PostMapping
     public String token(@RequestHeader(value = HttpHeaders.AUTHORIZATION) String headerContent) {
-        log.info("controller / token = " + headerContent);
-        /*Credentials credentialsObj = new Credentials(headerContent);
-        log.info(credentialsObj.getUsername());
-        log.info(credentialsObj.getPassword());*/
-        return "temp empty token string";
+       CredentialsDTO test = getCredentials(headerContent);
```
Comment (mukonin) [inline src/main/java/ua/volosiuk/mytokenservice/controller/TokenController.java:18]: pls check formatting (number of spaces)

File: src/main/java/ua/volosiuk/mytokenservice/controller/TokenController.java [hunk 3]
```diff
@@ -4,18 +4,18 @@
 import org.springframework.http.HttpHeaders;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import ua.volosiuk.mytokenservice.dto.CredentialsDTO;
+import static ua.volosiuk.mytokenservice.util.CredentialsUtils.getCredentials;
 
-@Log4j2
```
Comment (mukonin) [inline src/main/java/ua/volosiuk/mytokenservice/controller/TokenController.java:9]: pls check imports - remove outdated

File: src/main/java/ua/volosiuk/mytokenservice/controller/TokenController.java [hunk 4]
```diff
@@ -4,18 +4,18 @@
...
 @RestController
+@RequestMapping("/token")
 public class TokenController {
 
-    @PostMapping("/token")
+    @PostMapping
     public String token(@RequestHeader(value = HttpHeaders.AUTHORIZATION) String headerContent) {
```
Comment (mukonin) [inline src/main/java/ua/volosiuk/mytokenservice/controller/TokenController.java:17]: "value =" part is redundant, pls check RequestHeader source code

File: src/main/java/ua/volosiuk/mytokenservice/util/CredentialsUtils.java [hunk 5]
```diff
@@ -1,35 +1,33 @@
...
         }
-        return listCredentials;
     }
     private static String getDecodedString(String originalInput) {
         String cleanUserPass = originalInput.substring(6);
         byte[] decodedBytes = Base64.getDecoder().decode(cleanUserPass);
         return new String(decodedBytes);
     }
-}
+}
```
Comment (mukonin) [inline src/main/java/ua/volosiuk/mytokenservice/util/CredentialsUtils.java:33]: pls add empty line to the end of file

File: src/main/java/ua/volosiuk/mytokenservice/controller/TokenController.java (hunk 3 above)
Comment (RomanVolosiuk) [reply src/main/java/ua/volosiuk/mytokenservice/controller/TokenController.java:9]: clear. tnx

File: src/main/java/ua/volosiuk/mytokenservice/entity/User.java (hunk 1 above)
Comment (RomanVolosiuk) [reply src/main/java/ua/volosiuk/mytokenservice/entity/User.java:13]: done

File: src/main/java/ua/volosiuk/mytokenservice/controller/TokenController.java (hunk 4 above)
Comment (RomanVolosiuk) [reply src/main/java/ua/volosiuk/mytokenservice/controller/TokenController.java:17]: done

File: src/main/java/ua/volosiuk/mytokenservice/controller/TokenController.java (hunk 2 above)
Comment (RomanVolosiuk) [reply src/main/java/ua/volosiuk/mytokenservice/controller/TokenController.java:18]: done

File: src/main/java/ua/volosiuk/mytokenservice/util/CredentialsUtils.java (hunk 5 above)
Comment (RomanVolosiuk) [reply src/main/java/ua/volosiuk/mytokenservice/util/CredentialsUtils.java:33]: done

**Pull request merged**